module-2/backend/week_two.md

```diff
@@ -13,8 +15,21 @@ end
 ```
 
 What are some methods you can call on `Team`? If these methods aren't defined in the class, how do you have access to them?
+I can call find, group, select, where. 
+If the methods aren't defined in the class then I am not sure as how we have access to them.
```

Comment:
We have access because those methods are defined in ActiveRecord and we're inheriting from that.